### PR TITLE
test: 💍 Update test selectors for read hosts

### PR DESCRIPTION
**Pull request by calcaide**
# Description

Update test selectors for host-catalog -> read hosts.

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-15498)

## How to Test
- Run test suite successfully.
- In case we only want to validate the file changed: `(ui/admin)$: ember test --filter="Acceptance | host-catalogs | hosts | read"`

### CODE REVIEW

File: ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js [hunk 1]
```diff
@@ -87,26 +82,28 @@ module('Acceptance | host-catalogs | hosts | read', function (hooks) {
     instances.host.authorized_actions =
       instances.host.authorized_actions.filter((item) => item !== 'read');
 
-    await click(`[href="${urls.hosts}"]`);
-
-    assert.dom('.rose-table-body  tr:first-child a').doesNotExist();
+    await click(commonSelectors.HREF(urls.hosts));
+    assert.dom(commonSelectors.TABLE_FIRST_ROW_RESOURCE_LINK).doesNotExist();
```
Comment (cameronperera) [inline ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:86]: non-blocking: Could we add the space between the click and assert? 
```suggestion
    await click(commonSelectors.HREF(urls.hosts));
    
    assert.dom(commonSelectors.TABLE_FIRST_ROW_RESOURCE_LINK).doesNotExist();
```

File: ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js [hunk 2]
```diff
@@ -76,9 +72,8 @@ module('Acceptance | host-catalogs | hosts | read', function (hooks) {
     await a11yAudit();
     assert.strictEqual(currentURL(), urls.hosts);
 
-    await click(`[href="${urls.host}"]`);
+    await click(commonSelectors.HREF(urls.host));
     await a11yAudit();
-
```
Comment (cameronperera) [inline ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:81]: non-blocking: Could we add the space between the a11yAudit and assert?

**Review by cameronperera — APPROVED** [review]
LGTM. I left a couple of non-blocking comments.

**Review by DhariniJeeva — APPROVED** [review]
looks good!

**Review by ZedLi — APPROVED** [review]

**Pull request merged**